### PR TITLE
Update the kotlin version to  1.6.10 to avoid failing builds when use…

### PR DESCRIPTION
**Pull request by prasadsunny1**
…d in a flutter modules

This fixes https://github.com/fluttercommunity/flutter_sms/issues/70

### CODE REVIEW

**Comment by countryst** [comment]
When would this be released? Is there a way I can use this in my flutter application?

**Comment by prasadsunny1** [comment]
Yes @countryst, You can add add the kotlin version in your host project's build.gradle to temporarily make it build

**Comment by countryst** [comment]
```
  flutter_sms:
    git:
      url: https://github.com/fluttercommunity/flutter_sms.git
      ref: 419918ed46f1c4c675224546778a093e9fd09ca9 # commit hash
```
This worked https://stackoverflow.com/a/68784015/2603238

File: android/build.gradle [hunk 1]
```diff
@@ -2,10 +2,10 @@ group 'com.example.flutter_sms'
 version '1.0-SNAPSHOT'
 
 buildscript {
-    ext.kotlin_version = '1.3.50'
+    ext.kotlin_version = '1.6.10'
     repositories {
         google()
-        jcenter()
```
Comment (CrossBread) [inline android/build.gradle:8]: Is there a reason to switch to mavenCentral? This SO post suggests jcenter is more secure and offers better CDN performance.

https://stackoverflow.com/questions/50726435/difference-among-mavencentral-jcenter-and-mavenlocal

Comment (prasadsunny1) [reply android/build.gradle:8]: Here is an official note from android.
https://developer.android.com/studio/build/jcenter-migration

Jcenter is being deprecated in favour of maven central

Comment (CrossBread) [reply android/build.gradle:8]: Oof, thanks for the heads up.  This wound up being the root cause of some issues with some of my transient dependencies.

**Comment by prasadsunny1** [comment]
What do yo think about this PR?
is it good to go in?

**Comment by kmcgill88** [comment]
@rodydavis anyway there can be more contributors added to help get some of these prs merged?

**Comment by kvenn** [comment]
The fix was merged back in june. This can be closed.

https://github.com/fluttercommunity/flutter_sms/pull/81

**Pull request closed without merging**